packages/backend/lib/videos/videos.dto.d.ts

```diff
@@ -0,0 +1,12 @@
+export interface VideoDTO {
+    id: string;
+    title: string;
+    urlTrailer: string;
+    urlPhoto: string;
+}
```

Comment:
1. Pytanie: Widziałem w różnych materiałach, że faktycznie często na id dają string, ale pytanie do Ciebie czy do Damiana czy nie powinno być number (chyba że jest problem z konwersją?) 
2. Pytanie do tego samego co wyżej: Czy nie powinniśmy w jakiś sposób ustalić typów własnych jak np. Int poprzez własne utworzenie jakiegoś typu lub zaimportowanie biblioteki dedykowanej do tego? Albo czy potrzebujemy jakiegoś zabezpieczenia gdyby id było po przecinku np 1,045? 
A jeśli w ogóle nie bierzemy tego pod uwagę w ramach celów szkoleniowych to ok.